flink-cdc-common/src/main/java/org/apache/flink/cdc/common/source/DataSource.java

```diff
@@ -36,4 +37,18 @@ public interface DataSource {
     default SupportedMetadataColumn[] supportedMetadataColumns() {
         return new SupportedMetadataColumn[0];
     }
+
+    /**
+     * Indicating if this source guarantees for each TableId, it will not be evolved differently
+     * among subTasks. If returns {@code false}, you'll get a regular operator topology that is
+     * compatible with single-incremented sources like MySQL. Returns {@code true} for sources that
+     * does not maintain a globally sequential schema change events stream, like MongoDB or Kafka.
+     * <br>
+     * Note that new topology still an experimental feature. Return {@code false} by default to
+     * avoid unexpected behaviors.
+     */
+    @Experimental
+    default boolean canContainDistributedTables() {
```

Comment:
We use metadata concept in `DataSource` interface, `DistributedTables` is not accurate in my opinion, how about 
`boolean isParallelMetadataSource() `
or 
`boolean generatesMetadataInParallel()` ？

Reply:
Good point, I'll take the first one.